packages/frontend/src/hooks/cartesianChartConfig/useCartesianChartConfig.ts

```diff
@@ -18,13 +22,63 @@ type Args = {
     setPivotDimensions: React.Dispatch<
         React.SetStateAction<string[] | undefined>
     >;
+    columnOrder: string[];
+    explore: Explore | undefined;
+};
+
+export const sortDimensions = (
+    dimensionIds: string[],
+    explore: Explore | undefined,
+    columnOrder: string[],
+) => {
+    const getDimensionId = (dimension: Dimension) =>
+        `${dimension.table}_${dimension.name}`;
```

Comment:
We should reuse the `getItemId()` from common to avoid duplicating the id generation logic.